scripts/verify-issues.js

```
@@ -0,0 +1,122 @@
+  if (voteCounts.valid >= voteThreshold) {
+    newLabel = labelVerified;
+  } else if (voteCounts.duplicate >= voteThreshold) {
+    newLabel = labelDuplicate;
+  } else if (voteCounts.needsMoreInfo >= voteThreshold) {
+    newLabel = labelNeedsMoreInfo;
+  } else if (voteCounts.wontFix >= voteThreshold) {
+    newLabel = labelWontFix;
+  }
+
+  if (newLabel) {
+    await octokit.issues.update({
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)